kokoro/linux/build.sh

```diff
@@ -24,6 +24,7 @@ set -x
 
 BUILD_ROOT=$PWD
 SRC=$PWD/github/shaderc
+INSTALL_DIR="$SRC/install"
```

Comment:
Looks like this pollutes the source tree.  I'd put it beside the source tree, but under BUILD_ROOT.
Say, $BUILD_ROOT/install

Reply:
I put it beside the build dir.  We have:
$SRC/build
$SRC/install
If I move one I guess it makes sense to move both?

Reply:
Ah, I didn't see the part later in the script where the build dir was created and then cd'd into.

What you have is ok then.  It makes an "install" dir beside the build dir.